doc/CONTRIBUTING.md

```diff
@@ -0,0 +1,71 @@
+# WELCOME TO YOUR 
+# CONTRIBUTION GUIDELINES
+
+
+To help you save on time and hassle down the road, 
+these contribution guidelines will help for submitting well-formed pull requests and opening useful issues.
+
+For the repository owner, and contributors these guidelines are a way to communicate how people should contribute.
+
+The following is laid out to help govern and champion a productive collabrative effort.
+
+When deciding a title/name for your pull request please prefix with:
+a. (Feat): This is a new feature
+b. (Fix): I am fixing something
+c. (Doc): Some new documentation
+for example "fix.folder-structure" this outlines what you are doing and what you have done. it is short and sweet, to the point and informative. It will make documenting and revisiting code in the future easy to read and understand.
+
+For youthe contributor, reading these guidelines help us verify that they're submitting well-formed pull requests and opening useful issues.
+
+For both owners and contributors, contribution guidelines save time and hassle caused by improperly created pull requests or issues that have to be rejected and re-submitted.
+
+1. [How TO USE THE PULL REQUEST VIDEO EXPLAINATION] (https://youtu.be/rgbCcBNZcdQ) 
```

Comment:
If you want this to be a link you need to remove the space between the `]` and `(`. Same with all the links below

Reply:
All these suggestions were awesome thank you I did them all and @AffirmedVisionary the cheat sheet and other suggestions were super useful and @EbonyChantelle i didn’t realise I had accidentally created so many files in different places- hopefully I put most things right - ?